src/pages/attestering/Attestering.tsx

```diff
@@ -79,14 +75,20 @@ interface FormData {
     grunn?: string;
     begrunnelse?: string;
 }
-const Attestering = (props: Props) => {
-    const { sak, behandlingId } = props;
+const Attestering = () => {
     const [hasSubmitted, setHasSubmitted] = useState(false);
     const attesteringStatus = useAppSelector((s) => s.sak.attesteringStatus);
     const intl = useI18n({ messages });
     const dispatch = useAppDispatch();
+    const { sak } = useAppSelector((s) => ({ søker: s.søker.søker, sak: s.sak.sak }));
```

Comment:
Hva er grunnen til at `sak` ikke sendes inn som `prop` lenger?

Reply:
Den filen/react-componenten som sente in `sak` som prop er nå slettet!